backend/src/domain/models/SocketSession.js

```diff
@@ -20,7 +20,7 @@ export class SocketSession {
    * @param {*} name, users input nickname
    */
   addUser(socket, name) {
-    this.users[socket.id] = {socket: socket, name: name};
+    this.users.set(socket.id, {socket: socket, name: name});
```

Comment:
whats the difference between user.set and user[]?

Reply:
maybe code convention? not too sure either, but i believe this to be trivial so, merge or not idm

Reply:
it doesn't iterate when you use user[] for a map data structure